### PR TITLE
Files landing

### LINKED ISSUE #506: Files Landing Page

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #506 and contributes to #423 

@nataliejeremy can you review? We may want to get someone from CSE to review as well.

### CODE REVIEW

File: source/docs/articles/sites/files.md [hunk 1]
```diff
@@ -1 +1,12 @@
-draft: true
+---
+title: Pantheon Filesystem
+description: Learn how to work with the Pantheon filesystem.
+keywords: code, commit, sftp, files, filesystem, file location, directory location, directory, codebase
+---
+The Pantheon architecture is comprised of highly available [application containers](/docs/articles/architecture/all-about-application-containers/) that are seamlessly integrated with Valhalla, our cloud-based filesystem.
```
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: Let's lead with the platform definition of "files".  - the files directory, how to access via SFTP connection info, `cd files` or in client. It's important to note they're not local to the application container, are gitignored.

File: source/docs/articles/sites/files.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+
+Valhalla symbolically links the `wp-content/uploads` directory for WordPress and the `sites/default/files` directory for Drupal within the `files/` directory. Any [non-standard file locations](/docs/articles/sites/files/non-standard-files-locations/) should be symbolically linked to `/files` or manually moved.
+
+## Troubleshooting
+
+- [Filesystem Writeable Error](/docs/articles/sites/files/filesystem-faqs/)
```
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: This article has outstanding issues #352

**Comment by nataliejeremy** [comment]
+1 @ari-gold Can you review this please?

File: source/docs/articles/sites/files.md (hunk 2 above)
Comment (rachelwhitton) [inline source/docs/articles/sites/files.md]: Should I remove @bmackinney or just be aware of upcoming changes?

File: source/docs/articles/sites/files.md (hunk 1 above)
Comment (rachelwhitton) [inline source/docs/articles/sites/files.md]: Awesome! Thanks! .gitignored is important to mention here.

File: source/docs/articles/sites/files.md (hunk 2 above)
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: add Drupal to the link text

File: source/docs/articles/sites/files.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+Files are user uploads, usually images or documents. They are excluded from version control via Pantheon's .gitignore files:
+
+- [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore)
+- [WordPress](https://github.com/pantheon-systems/WordPress/blob/master/.gitignore)
+
+The Pantheon architecture is comprised of highly available [application containers](/docs/articles/architecture/all-about-application-containers/) that are seamlessly integrated with Valhalla, our cloud-based filesystem. This means that your files are not local to the application containers running your site's codebase.
+
+Valhalla symbolically links the `wp-content/uploads` directory for WordPress and the `sites/default/files` directory for Drupal within the `/files` directory. Any [non-standard file locations](/docs/articles/sites/files/non-standard-files-locations/) should be symbolically linked to `/files` or moved manually.
+
+## Access vis SFTP
```
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: typo

File: source/docs/articles/sites/files.md [hunk 4]
```diff
@@ -1 +1,21 @@
...
+While in SFTP mode, you can use framework admin tools (e.g. WordPress dashboard or Drupal's update manager) as well as command line tools for managing files.
```
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: This paragraph describes how to manage `code` not `files`. To manage files via SFTP, you dont need SFTP mode, just the connection info and a client. Can manage files right on Live if you want.

File: source/docs/articles/sites/files.md [hunk 5]
```diff
@@ -1 +1,21 @@
...
+
+- [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore)
+- [WordPress](https://github.com/pantheon-systems/WordPress/blob/master/.gitignore)
+
+The Pantheon architecture is comprised of highly available [application containers](/docs/articles/architecture/all-about-application-containers/) that are seamlessly integrated with Valhalla, our cloud-based filesystem. This means that your files are not local to the application containers running your site's codebase.
+
+Valhalla symbolically links the `wp-content/uploads` directory for WordPress and the `sites/default/files` directory for Drupal within the `/files` directory. Any [non-standard file locations](/docs/articles/sites/files/non-standard-files-locations/) should be symbolically linked to `/files` or moved manually.
+
+## Access via SFTP
+While in SFTP mode, you can use framework admin tools (e.g. WordPress dashboard or Drupal's update manager) as well as command line tools for managing files.
```
Comment (bmackinney) [inline source/docs/articles/sites/files.md]: kill this line

**Pull request merged**